src/routes/transactions/entities/__tests__/human-description.builder.ts

```diff
@@ -0,0 +1,42 @@
+import { Builder, IBuilder } from '@/__tests__/builder';
+import {
+  RichAddressFragment,
+  RichTokenValueFragment,
+  RichTextFragment,
+  RichDecodedInfoFragment,
+  RichFragmentType,
+} from '@/routes/transactions/entities/human-description.entity';
+import { faker } from '@faker-js/faker';
+
+function richTokenValueFragmentBuilder(): IBuilder<RichTokenValueFragment> {
+  return Builder.new<RichTokenValueFragment>()
+    .with('type', RichFragmentType.TokenValue)
+    .with('value', faker.number.int().toString())
```

Comment:
```suggestion
    .with('value', faker.string.numeric())
```

See https://fakerjs.dev/api/string.html#numeric